userprofile/migrations/0005_auto_20211121_1928.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.2.7 on 2021-11-22 00:28
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('userprofile', '0004_auto_20211109_0003'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='profile',
+            old_name='adultBaseball',
+            new_name='baseball',
+        ),
+        migrations.RenameField(
+            model_name='profile',
+            old_name='adultFootball',
+            new_name='football',
+        ),
+        migrations.RenameField(
+            model_name='profile',
+            old_name='adultSoftball',
+            new_name='softball',
+        ),
+        migrations.RemoveField(
+            model_name='profile',
+            name='littleLeagueBaseball',
+        ),
+        migrations.RemoveField(
+            model_name='profile',
+            name='littleLeagueSoftball',
+        ),
+        migrations.RemoveField(
+            model_name='profile',
+            name='tBall',
+        ),
+        migrations.RemoveField(
+            model_name='profile',
+            name='youthFootball',
+        ),
+        migrations.AddField(
```

Comment:
We don't currently have a map for track, so I am wondering if we should include it as an option here? Let me know what you think, otherwise, it looks great!